build/makelib/golang.mk

```diff
@@ -49,7 +49,7 @@ GO_TEST_FLAGS ?=
 # ====================================================================================
 # Setup go environment
 
-GO_SUPPORTED_VERSIONS ?= 1.7|1.8|1.9|1.10|1.11
+GO_SUPPORTED_VERSIONS ?= 1.7|1.8|1.9|1.10|1.11|1.12
```

Comment:
Do we even support older than 1.11? Seems like when we moved to 1.11 we haven't been able to build with 1.10 since then.

Reply:
@travisn So should I remove all versions older than 1.11 in this PR?

Reply:
@galexrt yes, sounds good